File: 240116/배열의 특정 위치 찾기/find-specific-location-fo-array.c
#include <stdio.h>

int main() {
    // 여기에 코드를 작성해주세요.
    int cnt=0;
    int sum=0;
    int sum_a=0;
    int arr[10];
    for (int i=0;i<=9;i++){
        scanf("%d",&arr[i]);
        if (i%2!=0){
            sum+=arr[i];
        }
        else if ((i+1)%3==0){
            cnt+=1;
            sum_a+=arr[i];
        }
    }
    printf("%d %.1lf",sum,(double)sum_a/cnt);
    return 0;
}

File: 240314/배열의 특정 위치 찾기/find-specific-location-fo-array.java
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        // 여기에 코드를 작성해주세요.
        Scanner sc=new Scanner(System.in);
        int[] arr=new int[10];
        int sum=0;double cnt=0;
        int sums=0;
        for (int i=0;i<10;i++){
            arr[i]=sc.nextInt();
            if ((i+1)%2==0){
                sum+=arr[i];
            }
            if ((i+1)%3==0){
                sums+=arr[i];
                cnt+=1;
            }
        }
        System.out.printf("%d %.1f",sum,sums/cnt);
    }
}